onewallet/src/main/java/org/kinecosystem/onewallet/OneWalletClient.kt

```diff
@@ -0,0 +1,111 @@
+package org.kinecosystem.onewallet
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Handler
+import android.os.Looper
+import android.util.Log
+import android.widget.Toast
+import kin.sdk.KinAccount
+import org.kinecosystem.common.base.LocalStore
+import org.kinecosystem.onewallet.presenter.LinkWalletPresenter
+import org.kinecosystem.onewallet.view.LinkWalletViewHolder
+import org.kinecosystem.transfer.sender.manager.TransferManager
+import java.util.concurrent.Executors
+
+class OneWalletClient : IOneWalletClient {
+    companion object {
+        val ONE_WALLET_APP_ID = "org.kinecosystem.kinit"
```

Comment:
I understand that the one wallet is bounded currently to kinit only?
we'll you have also the other side (kinit) APIs currently or at a later stage?

Reply:
At the moment it's kinit only